Readme.md

```diff
@@ -1,4 +1,71 @@
-## Overview
+## Copyright notice
+
+While the code itself is MIT licensed, the coding challenge itself, and its description,
+included in this `Readme.md` file belongs to Agile Freaks et al.
+
+
+## Coffee Place 
+
+This is an attempt at solving the closest coffee shop coding challenge. 
+
+All good software starts with making flawed assumptions.  
```

Comment:
:100:

Reply:
If that ain't the truth, I don't know what is! :joy_cat: